tests/eventhub_functions/eventhub_trigger/function.json

```diff
@@ -0,0 +1,23 @@
+{
+  "scriptFile": "__init__.py",
+  "disabled": false,
+
+  "bindings": [
+    {
+      "type": "eventHubTrigger",
+      "name": "event",
+      "direction": "in",
+      "cardinality": "many",
```

Comment:
eventHubName is missing. Here is a sample function.json
```
      "type": "eventHubTrigger",
      "name": "myEventHubMessage",
      "direction": "in",
      "eventHubName": "samples-workitems",
      "connection": "AppSettingName",
      "consumerGroup": "$Default"
```

Reply:
I tried that. With `eventHubName` I get:

```
fail: Host.Startup[0]
      Error indexing method 'Functions.eventhub_trigger'
Microsoft.Azure.WebJobs.Host.Indexers.FunctionIndexingException: Error indexing method 'Functions.eventhub_trigger' ---> System.InvalidOperationException: No event hub receiver named python-worker-ci
   at Microsoft.Azure.WebJobs.ServiceBus.EventHubConfiguration.GetEventProcessorHost(String eventHubName, String consumerGroup) in C:\projects\azure-webjobs-sdk-rqm4t\src\Microsoft.Azure.WebJobs.Extensions.EventHubs\EventHubConfiguration.cs:line 290
```

Reply:
You need to connect to an existing event hub. Here are the instructions : [event-hubs-create](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create). When you create event hub trigger from portal, UI helps with choosing an existing eventhub or creating a new one.

Reply:
I've done that. I have the connection string and everything. Doesn't help.

Reply:
I see. I will set up a C# event hug trigger example with the python-worker-ci eventhub and get back to you.

Reply:
I quickly checked with the C# and JavaScript examples. It seems that `path` and `eventHubName` have been used interchangeably to represent name of the event hub. See here: [event hub configuration](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-event-hubs#trigger---configuration)

Reply:
Adding @mhoeger to confirm while Pragna is out.

Reply:
@elprans  - Created EventHubTriggerCSharpTest in python-worker-ci-appveyor app. Here is the sample to trigger the function: https://github.com/soninaren/EventhubTrigger . Update eventHubName and connectionstring before running the sample

Reply:
@pragnagopa Is there a working `function.json` example?

Reply:
The `EventHubTriggerCSharpTest` still has `path` in its definition, which doesn't work with the latest host/extension:

```json
{
  "bindings": [
    {
      "type": "eventHubTrigger",
      "name": "myEventHubMessage",
      "direction": "in",
      "path": "pythonworkereventhubci",
      "connection": "pythonworkereventhub_RootManageSharedAccessKey_EVENTHUB",
      "consumerGroup": "$Default",
      "cardinality": "many"
    }
  ],
  "disabled": false
}
```

Reply:
Can you check EventHubTriggerCSharpTest ? Switch to Advanced editor for function.json

Reply:
Can you try
- change path to eventHubName
- Add app setting key: pythonworkereventhub_connection value: get connection string from the eventhub

Reply:
How do I pass the app settings to the local host instance?

Reply:
set environment variable